pylint/test/functional/mutable_sequence_modified_in_loop.py

```diff
@@ -0,0 +1,66 @@
+""" The mutable sequence was modified inside the loop """
+
+# pylint:disable=print-statement, missing-docstring, redefined-outer-name, invalid-name, using-constant-test, syntax-error, no-member, superfluous-parens
```

Comment:
Why `syntax-error` is ignored? Your syntax errors are related to print statements, use `from __future__ import print_function` and this will allow you to have same test for both python2 and python3. Right now there's no coverage for whole Python 3 family.

`superfluous-parens` and `print-statement` may also be not needed?